src/pages/en/guides/markdown-content.md

```diff
@@ -33,7 +33,7 @@ Markdown does support **bold** and _italics._
 To learn more about adding a layout to your page, read the next section on **Markdown Layouts.**
 ```
 
-### Markdown Layouts
+### Frontmatter `layout:`
```

Comment:
```suggestion
### Frontmatter `layout`
```

I'd probably remove the trailing colon as it's not part of the property name itself, and it looks unusual both in the heading and the sidebar.